pkg/postage/listener/listener.go

```diff
@@ -149,6 +152,8 @@ func (l *listener) Listen(from uint64, updater postage.EventUpdater) <-chan stru
 		cancel()
 	}()
 
+	chainUpdateInterval := time.Duration(l.blockTime) * pollEvery * time.Second
```

Comment:
i don't think this formula makes sense. why would the update interval increase if tailsize goes up? it should either be `time.Duration(l.blockTime)` or `time.Duration(l.blockTime) / constantFactor`.